build_torch_xla_libs.sh

```diff
@@ -26,7 +26,9 @@ if [[ "$XLA_CUDA" == "1" ]] && [[ "$CLOUD_BUILD" == "true" ]]; then
 fi
 
 OPTS=(--cxxopt="-std=c++14")
-if [[ "$CC" =~ ^clang ]]; then
+
+${CC} --version | grep 'clang' &> /dev/null
```

Comment:
Hmm ... why?

Reply:
Say if CC contains a full path to clang binary, previous check would not work since it requires the string to start with "clang".

Reply:
`basename $CC` feels better then.

Reply:
Sounds good. Done. PTAL.